### DISCUSSION TITLE
SEO Pro: Meta description from bard field

### DISCUSSION THREAD

**Discussion by markussorg** [statamic/cms#5387] [Q&A]
Hey,

I have the following problem with the SEO Pro Addon: I want to populate the meta description field from a bard field. The bard field is within a fieldset "Bard Editor". How do I reference this bard field in the meta description? Is this even possible? Something like bard_editor/bard is not working!

Thanks for your support!

**Content:**

![image](https://user-images.githubusercontent.com/3519464/156362535-259c4071-9ad4-4353-acc1-84fb1d22fbad.png)

**Fieldset**

![image](https://user-images.githubusercontent.com/3519464/156362621-c41f2c77-2a3f-43ce-9245-8ce10343bcd0.png)

**Meta description**

![image](https://user-images.githubusercontent.com/3519464/156363685-44ac53fb-d6a5-4693-9723-cbe63ede7353.png)

**Comment by edalzell** [comment]
Ya I suspect that won't work. Bard is an array field I bet those can't be pulled in like that.
